xdrip-get-entries.sh

```diff
@@ -1087,6 +1095,12 @@ function process_delta()
     da=$(bc <<< "0 - $da")
   fi
 
+# EYF try
```

Comment:
This new code is marked as `try`, so just wanted to make sure you intended to merge it with the oref0 plugin change for fakemeter.

Reply:
Good catch - I will remove that comment and check the corresponding code

Reply:
I removed the code - you are right it was not intended to be included. It was something I was trying to halve the bg differential in cases of high noise. It didn't really help much - sensor was dying anyway ... better to see noise as it is.